static/src/javascripts/projects/commercial/modules/dfp/dfp-api.spec.js

```diff
@@ -498,7 +513,7 @@ describe('DFP', () => {
                 callback(tcfv2WithConsent)
             );
             getConsentFor.mockReturnValue(true);
-            prepareGoogletag().then(() => {
+            return prepareGoogletag().then(() => {
```

Comment:
asynchronous Jest tests [should return the Promise][1], otherwise they can fail silently.

[1]: https://jestjs.io/docs/en/tutorial-async

```js
// The assertion for a promise must be returned.
```